internal/serverinstall/ecs.go

```diff
@@ -495,6 +494,24 @@ func (i *ECSInstaller) Upgrade(
 		return nil, fmt.Errorf("no waypoint-server service found")
 	}
 
+	// Set or update the deregistration delay in the target groups. Waypoint
+	// server installations pre-0.9.1 used the default 300 second delay, which
+	// we need to lower. Note: "serverSvc.LoadBalancers" below is a slice of
+	// load balancer + target group pairs. We expect two target groups attached
+	// to the same load balancer.
+	// See
+	// https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_DescribeServices.html
+	for _, lb := range serverSvc.LoadBalancers {
+		if lb.TargetGroupArn != nil {
+			s.Update("Updating deregistration delay and termination for Target Groups")
+			if err := modifyTargetGroups(elbv2.New(sess), *lb.TargetGroupArn); err != nil {
+				s.Update("Error updating Target Group %s: %w", *lb, err)
```

Comment:
Do we need to return the `err` here? There's a couple of other places where we don't, is that intentional? And should we preserve the error in a `UI.Output` or maybe a `log.Error`? Otherwise the next stepgroup will wipe over this output.